bin/duo-install

```diff
@@ -0,0 +1,339 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
```

Comment:
You're probably right, I'll extract some of those helper fns into a utils file or something.